edagames_grpc/server.py

```diff
@@ -0,0 +1,95 @@
+import grpc
+import eda_games_pb2
+import eda_games_pb2_grpc
+from .game_state import GameState
+from typing import Dict, List
+
+
+def prepare_game_state(
+    game_state: GameState
+) -> eda_games_pb2.GameStateResponse:
+    response = eda_games_pb2.GameStateResponse()
+    response.current_player = game_state.current_player
+    response.next_player = game_state.next_player
+    response.turn_data.update(game_state.turn_data)
+    response.play_data.update(game_state.play_data)
+    return response
+
+
+class EdaGamesService(eda_games_pb2_grpc.EdaGameServiceServicer):
+    def __init__(self, delegate):
```

Comment:
NIT: Add the delegate's type if possible

Reply:
Added in 41a16fc